src/groundwater/aquifer.jl

```diff
@@ -111,12 +111,15 @@ instead. Specific yield will vary roughly between 0.05 (clay) and 0.45 (peat)
 """
 @get_units struct UnconfinedAquifer{T} <: Aquifer
     head::Vector{T} | "m"  # hydraulic head [m]
-    k::Vector{T} | "m d-1"  # horizontal conductivity [m d⁻¹]
+    kh₀::Vector{T} | "m d-1"  # reference horizontal conductivity [m d⁻¹]
     top::Vector{T} | "m" # top of groundwater layer [m]
     bottom::Vector{T} | "m" # bottom of groundwater layer
     area::Vector{T} | "m2"
     specific_yield::Vector{T} | "m m-1" # [m m⁻¹]
     conductance::Vector{T} | "m2 d-1" #
+    exp_conducitivity::Bool | "-" # True or False wether to use exponential decay of conductivity
```

Comment:
Incorrect spelling of `exp_conductivity`, and perhaps leave this out the Struct? We do not define this for a confined aquifer, and the line 317 `flux!(Q, gwf.aquifer, gwf.connectivity, gwf.aquifer.exp_conductivity)` will not work in that case.

Reply:
Thanks for spotting the typo! Fixed this, and also moved the flag outside of the struct. All functions should now work for both UnconfinedAquifer and ConfinedAquifers